lib/sgtn-client/api/translation.rb

```diff
@@ -56,6 +58,7 @@ def self.getString_f(component, key, args, locale, *optionals)
 
       def self.getStrings(component, locale)
         SgtnClient.logger.debug "[Translation][getStrings]component=#{component}, locale=#{locale}"
+        locale = SgtnClient::LocaleUtil.get_best_locale(locale)
```

Comment:
If locale is source locale, return source directly.
```suggestion
        locale = SgtnClient::LocaleUtil.get_best_locale(locale)
        return getSources(component, locale) if locale == LocaleUtil.get_source_locale

```

Reply:
get_best_locale will replace 'en' with SgtnClient::Config.configurations.default so the source bundle will be loaded.